src/display/editor/drawers/outline.js

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2023 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class Outline {
+  /**
+   * @returns {string} The SVG path of the outline.
+   */
+  toSVGPath() {
+    throw new Error("Abstract method `toSVGPath` must be implemented.");
```

Comment:
Nit: Here, and below in this file, since we're in the `src/`-folder we could use an existing helper instead (compare with e.g. [src/core/base_stream.js](https://github.com/mozilla/pdf.js/blob/master/src/core/base_stream.js)).
```suggestion
    unreachable("Abstract method `toSVGPath` must be implemented.");
```